src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -64,6 +65,12 @@ public ProcessMetrics(ProcessInstrumentationOptions options)
             },
             unit: "By",
             description: "The amount of virtual memory allocated for this process that cannot be shared with other processes.");
+
+        this.MeterInstance.CreateObservableCounter(
+            "process.cpu.time",
+            () => this.GetProcessorTimes(),
```

Comment:
We will have to do similar thing here as is in other instruments because of this: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/664#issue-1388565952.

```csharp
{
    if (!this.virtualMemoryUsage.HasValue)
    {
        this.Snapshot();
    }

    var value = this.virtualMemoryUsage.Value;
    this.virtualMemoryUsage = null;
    return value;
}

```